recipes/rgi/3.2.1/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Darwin" ]; then
```

Comment:
These echos shouldn't be needed for conda recipes. You should be fine with just the `$PYTHON setup.py install`.